src/Interfaces/LazySet.jl

```diff
@@ -878,3 +879,93 @@ true
 ```
 """
 isconvextype(X::Type{<:LazySet}) = false
+
+"""
+    area(X::LazySet{N}) where {N}
+
+Compute the area of a set.
+
+### Input
+
+- `X` -- set
+
+### Output
+
+A number representing the area of `X`.
+"""
+function area(X::LazySet{N}) where {N}
+    if dim(X) == 2
+        return area_2D(X)
+    else
+        throw(ArgumentError("the area function only applies to two-dimensional sets, " *
+                            "but the given set is $(dim(X))-dimensional"))
+    end
+end
+
+"""
+    area_2D(X::LazySet{N}) where {N}
+
+Compute the area of a two-dimensional polytopic set using the Shoelace formula.
+
+### Input
+
+- `X` -- two-dimensional set
+
+### Output
+
+A number representing the area of `X`.
+
+### Algorithm
+
+Let `m` be the number of vertices of `X`. The following instances are special-cased:
+
+- `m = 0, 1, 2`: the output is zero.
+- `m = 3`: the triangle case is computed using the Shoelace formula with 3 points.
+- `m = 4`: the quadrilateral case is obtained by the factored version of the Shoelace
+           formula with 4 points.
+
+Otherwise, the general Shoelace formula is used; for detals see the wikipedia article
+[Shoelace formula](https://en.wikipedia.org/wiki/Shoelace_formula).
+"""
+function area_2D(X::LazySet{N}) where {N}
+    @assert dim(X) == 2 "this function only applies to two-dimensional sets, " *
+    "but the given set is $(dim(X))-dimensional"
+
```

Comment:
I would not check this again. Or remove the other method completely.

Reply:
the change you propose is connected to the previous thread, https://github.com/JuliaReach/LazySets.jl/pull/2062#discussion_r391646940 . i can make the proposed change now, about adding a `surface` function. the top-level `area` should also check that `X` is two-dimensional -- so it's computed twice if the user calls `surface(p)`. but this shouldn't be a problem, since that operation is super cheap.

Reply:
The `surface` function did not exist back then (I think). With the new structure I agree to test it twice.

Reply:
ok!